concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -268,7 +268,7 @@ Desrochers-2010-330-266	266	esparcir	spread	SPREAD OUT	38	Verb	3.7843137254902	8
 Desrochers-2010-330-267	267	fustigar	whip			Verb	2.77450980392157	8	5	
 Desrochers-2010-330-268	268	garabatear	scribble			Verb	3.38235294117647	10		
 Desrochers-2010-330-269	269	guardar	protect	PROTECT	2900	Verb	6.2843137254902	7	116	6.76
```

Comment:
I'd unmap `Desrochers-2010-330-269	269	guardar	protect	PROTECT	2900`. It means "to save something".